templates/deployments/appname/app.jsonnet.tpl

```diff
@@ -62,7 +62,11 @@ local all = {
 	namespace: ok.Namespace(app.namespace) {
 		metadata+: {
 			annotations+: {
-				'iam.amazonaws.com/permitted': '%s_service_role' % app.name,
+        {{- if not (stencil.Arg "aws.useKIAM") }}
```

Comment:
issue: generally avoid a conditional with a negative check, when you have branches for both conditions.

Reply:
Will reverse this